package.json

```diff
@@ -18,7 +18,7 @@
     "vscode": "^1.43.0"
   },
   "dependencies": {
-    "@zowe/cli": "7.0.0-next.202109281609",
```

Comment:
Thanks for updating.

Since the CLI is being more careful when making schema changes to a vNext version, do we want to consider automating this kind of PRs ?
Just a thought 😋